### ISSUE TITLE
Covert to Doom theme

### ISSUE THREAD

**Issue by NamesCode** [catppuccin/emacs#32]

**Comment by NamesCode** [comment]
Work has started.

**Comment by ethangoan** [comment]
Hi @NamesCode , I actually started on a doom port a while ago for the mocha and latte themes. It is pretty sloppy, but might be able to help you get the ball rolling. Happy to help develop it with you as well if you want to start a clean repo of it.
https://github.com/ethangoan/doom-themes/tree/catppuccin

**Comment by NamesCode** [comment]
Thanks for the offer but I had contacted @mangkoran a few days ago before you commented this and already used his code as a base found here: https://github.com/mangkoran/themes/tree/doom-catppuccin Thanks for your work clearing up some of the issues on the non-doom themes :).

**Comment by ethangoan** [comment]
no worries, this looks like a nicer and more succinct solution to what I was working on :)

**Comment by NamesCode** [comment]
Yeah my bad. I should've done that at the start. I'm new to things like GitHub.

**Comment by NamesCode** [comment]
Quick question? What's your email?

**Comment by mangkoran** [comment]
It is `afnazrie@gmail.com`. If you use Git CLI, you could put the following line, [spaced 2 lines](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors) after the commit message.
```
Co-authored-by: Afiq Nazrie <afnazrie@gmail.com>
```

**Comment by NamesCode** [comment]
My git CLI is broken at the moment so im gonna use github. Should be coauthored in a min

**Comment by NamesCode** [comment]
I will have to attempt later when I can access my main device to try and fix git to do this.

**Comment by mangkoran** [comment]
No worries at all. Take your time and hope you enjoy learning Git and GitHub!

**Comment by mangkoran** [comment]
If you use GitHub web, you could put the co-author at the bottom of the commit message, with 2 space/newline gap.
![image](https://user-images.githubusercontent.com/57625126/206143540-7aaf97d4-20ac-48b1-a12b-25b60cbc41fd.png)

**Comment by mangkoran** [comment]
Also it's okay. We can move on from this. No need to further add me as co-author thingy.

**Comment by mangkoran** [comment]
Next time, you should've use pull request instead of committing directly to master 👍

**Comment by mangkoran** [comment]
Uh could you reset and force-push the main branch? 😅
![image](https://user-images.githubusercontent.com/57625126/206145385-290233e1-60a6-4650-89de-0559b31aa9d3.png)

**Comment by NamesCode** [comment]
Yeah. at home now and fixed git

**Comment by NamesCode** [comment]
Done

**Comment by ethangoan** [comment]
@NamesCode @mangkoran are you planning on making a pull request to have it added to the doom themes repo?

Also if you want a hand developing on it let me know, might be able to help test and tweak a few things if needed :)

**Comment by mangkoran** [comment]
@ethangoan I have opened a PR here https://github.com/doomemacs/themes/pull/760

Additional help are welcome!
